Quotation.Rmd

```diff
@@ -272,7 +272,7 @@ There are two other important base quoting functions that we'll cover elsewhere:
 ### Substitution
 \indexc{substitute()}
 
-You'll most often see `substitute()` used to capture unevaluated arguments. However, as well as quoting, `substitute()` also does "substitution": if you give it an expression, rather than a symbol, it will substitute in the values of symbols defined in the current environment. 
+You'll most often see `substitute()` used to capture unevaluated arguments. However, as well as quoting, `substitute()` also handles substitution. If you give it an expression, rather than a symbol, it will substitute in the values of symbols defined in the current environment. 
```

Comment:
```suggestion
You'll most often see `substitute()` used to capture unevaluated arguments. However, as well as quoting, `substitute()` also does substitution (as its name suggests!). If you give it an expression, rather than a symbol, it will substitute in the values of symbols defined in the current environment. 
```